packages/app-desktop/integration-tests/wcag.spec.ts

```diff
@@ -55,10 +55,6 @@ test.describe('wcag', () => {
 
 		await mainScreen.createNewNote('Test');
 
-		// For now, activate all notes to make it active. When inactive, it causes a contrast warning.
-		// This seems to be allowed under WCAG 2.2 SC 1.4.3 under the "Incidental" exception.
```

Comment:
This doesn't seem to be true &mdash; the "incidental" exception includes an exception for "inactive" user interface components, but this means that the component is "not available for user interaction", which is not the case for "all notes" (see ["key terms"](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#key-terms)).